packages/react/ds/src/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,43 @@
+import { Link } from '../link/link.js';
+import type { BreadcrumbLinkProps, BreadcrumbProps } from './types.js';
+
+export const BreadcrumbEllipsis = () => (
+  <div className="gi-breadcrumb-ellipsis" aria-hidden="true">
```

Comment:
suggestion: Not sure if you have consult this with Lexa or not, but we have an icon for this.

```
<span data-testid="govie-icon" class="material-symbols-outlined gi-gray-700">more_horiz</span>
```

Reply:
hmm I didn't know that, let me check it.

Reply:
I have used in my html pagination PR if you want a reference.